src/solvers/refinement/string_refinement.cpp

```diff
@@ -1076,8 +1076,10 @@ std::map<exprt, int> string_refinementt::map_representation_of_sum(
     to_process.pop_back();
     if(cur.id()==ID_plus)
     {
-      to_process.push_back(std::make_pair(cur.op1(), positive));
-      to_process.push_back(std::make_pair(cur.op0(), positive));
+      for(const exprt &op : cur.operands())
```

Comment:
@peterschrammel for my own curiosity, why would you prefer `auto` here? I find `exprt` more informative to the reader